docs/Quickstart.md

```diff
@@ -17,71 +19,85 @@ $ git commit -m 'new rails app'
+4. Add ShopifyApp to Gemfile
 ----------------------------
+
+Run these commands to add the `shopify_app` Gem to your app:
+
 ```sh
 $ echo "gem 'shopify_app'" >> Gemfile
 $ bundle install
 ```
 
-Note - it's recommended to use the latest version of the Shopify Gem. Check the Git tags to see the latest release and then add it to your Gemfile e.g `gem 'shopify_app', '~> 7.0.0'`
+**Note:** we recommend using the latest version of Shopify Gem. Check the [Git tags](https://github.com/Shopify/shopify_app/tags) to see the latest release version and then add it to your Gemfile e.g `gem 'shopify_app', '~> 7.0.0'`
 
 5. Run the ShopifyApp generator
 -------------------------------
+
+Generate the code for your app by running these commands: 
+
 ```sh
-# use the keys from your app in the partners area
+# Use the keys from your app you created in the partners area
 $ rails generate shopify_app --api_key <shopify_api_key> --secret <shopify_api_secret>
 $ git add .
 $ git commit -m 'generated shopify app'
 ```
 
 If you forget to set your keys or redirect uri above, you will find them in the shopify_app initializer at: `/config/initializers/shopify_app.rb`.
 
-We recommend adding a gem or utilizing ENV variables to handle your keys before releasing your app.
+We recommend adding a gem or utilizing environment variables (ENV) to handle your keys before releasing your app. [Learn more about using environment variables.](https://www.honeybadger.io/blog/ruby-guide-environment-variables/)
 
 6. Deploy your app
 ---------
+
+Once you've generated your app, push it into your Heroku environment to see it up and running:
 ```sh
 $ git push heroku
 $ heroku run rake db:migrate
 ```
 
 7. Install the App!
 -------------------
- Ensure you have created a development store on the Shopify Partner Dashboard. If you don't already have one, [create one by following these instructions](https://help.shopify.com/en/api/getting-started/making-your-first-request#create-a-development-store).
+
+Ensure you have created a [development store](https://help.shopify.com/en/api/getting-started/making-your-first-request#create-a-development-store) using the Shopify Partner Dashboard. If you don't already have one, [create one by following these instructions](https://help.shopify.com/en/api/getting-started/making-your-first-request#create-a-development-store).
 
 ##### Note: The following step will cause your store to become `transfer-disabled.` Read more about store transfer and why it's important [here](https://help.shopify.com/en/api/guides/store-transfers#transfer-disabled-stores). This is an irreversible change, so be sure you don't plan to transfer this store to a merchant.
 
-Install the app using the Partner Dashboard in the Apps area.
+Install the app onto your new development store using the Partner Dashboard by heading to the apps page, clicking your app, and looking for the `Test your app` instructions where you can select a store to install your app on.
```

Comment:
"heading to" is an English idiom, we could just say "opening the apps page"

(if only we could link directly to their apps index!)